src/lib/init.ts

```diff
@@ -10,14 +12,31 @@ import {
   EMBEDDED_POPOVER_CLASS,
   EMBEDDED_POPOVER_CLOSE_CLASS,
 } from "../utils/iframe";
+import { showMarketplaceDefaults } from "./showMarketplace";
 
 export interface InitProps
   extends Pick<State, "screenConfiguration" | "theme" | "translation">,
     Partial<Pick<State, "filters" | "prismaticUrl">> {}
 
-export const init = (options?: InitProps) => {
+const optionsDefault = {
+  filters: {
+    ...showMarketplaceDefaults["filters"],
```

Comment:
Let's move all the defaults here since they are defaults for every subsequent method call.